symbolic-symcache/tests/test_writer.rs

```diff
@@ -109,3 +110,51 @@ fn test_write_large_symbol_names() -> Result<(), Error> {
 
     Ok(())
 }
+
+/// This tests the fix for the bug described in
+/// https://github.com/getsentry/symbolic/issues/284#issue-726898083
+#[test]
+fn test_lookup_no_lines() -> Result<(), Error> {
+    let buffer = ByteView::open(fixture("xul.sym"))?;
+    let object = Object::parse(&buffer)?;
+
+    let mut buffer = Vec::new();
+    SymCacheWriter::write_object(&object, Cursor::new(&mut buffer))?;
+    let symcache = SymCache::parse(&buffer)?;
+    let symbols = symcache.lookup(0xc6dd98)?.collect::<Vec<_>>()?;
+
+    assert_eq!(symbols.len(), 1);
+    let name = symbols[0].function_name();
+
+    assert_eq!(
+        name.try_demangle(DemangleOptions::name_only()),
```

Comment:
Is there a particular reason you're demangling the identifier? Sometimes, updates to the demangler cause subtle changes in the resulting demangled function names, which would then break this test.

In this case, the symbol you're feeding in is already demangled (Breakpad symbol). You can just assert the name directly. As a nice side effect, we get rid of the dependency to `symbolic-demangle`.